### ISSUE TITLE
Code formating not properly working on HTML

### ISSUE THREAD

**Issue by bastienchassagnol-servier** [rjournal/rjtools#48]
In RJwrapper.tex is provided a \code{R_function} command, that enabled in previous purely Latex styles to emphasize specific R functions. However, it seems that this option has not been adjusted to the new Rmd format, as it's only working for the Latex ouput (the text within code brackets does not even appear in the HTML version).

It seems that enabling the [downlit](https://downlit.r-lib.org/) personalization, as suggested by the distill documentation, would be a rather nice evolution to highlight these tiny pieces of code, wouldn't it?

**Comment by dicook** [comment]
Hi @bastienchassagnol-servier would you have an example for how this might work? 
Right now code should be delimited by backticks, not using the \code{}, but happy for making it look better. It would need to work for both html and pdf outputs.

**Comment by bastienchassagnol-servier** [comment]
Hi @dicook, 
I carried out some experiences to check the behaviour of downlit library.
With the current rjtools template, you simply need  to load library *downlit* in the setup chunk, to automatically highlight chunks of R code.

However, for inline code formatting, you need to explicitly call either function downlit::autolink("R_function") to highlith a R function, or href_topic or href_package to automatically generate a web link towards the corresponding vignette or CRAN reference webpage.

Here's the code used for that purpose, using the same YAML preamble as provided by rjtools package:


```{r plot-example, layout="l-body-outset", fig.width=6, fig.height=1.5, echo=TRUE}
library(ggplot2)
ggplot(mtcars, aes(hp, mpg)) + geom_point() + geom_smooth()
```

The `r downlit::autolink("stats::median()")` is a function highlited using the `r downlit::href_package("downlit")` package // `r downlit::autolink("library(downlit)")` package // `downlit` package. It's also possible to automatically link to a specific topic:
`r downlit::autolink('vignette("mclust", package = "mclust")')` or `r href_topic("mclust", "mclust")`.`

And the corresponding HTML output:
<img width="425" alt="image" src="https://user-images.githubusercontent.com/82661205/186644934-3eb46f91-b572-4699-9a6d-8ae33714bf3f.png">
 (you can notice that additionaly to the formatting of inline piece of code, by dragging the mouse over it, an automatic link to the corresponding documention has been generated). Unfortunately, this doesn't work as well with the PDF output:
<img width="329" alt="image" src="https://user-images.githubusercontent.com/82661205/186645048-4409d634-3ae0-42b6-a46b-2a50a31bd6ff.png">

To sum it up, downlit is a relevant tool to highlight pieces of code, but lacks in my opinion, two key features:

- the formatting of the vignette links is not really comfortable (the whole HTML link is used, instead of a smaller proxy, such as the title of the vignette)
- the PDF output is partly broken

Maybe, you may develop a similar device as command \CRANpkg, but that would enable to highlight additionally inline and multiline R functions and blocks + enable automated redirection to the specific vignette of a package.
